pyproject.toml

```diff
@@ -10,6 +10,7 @@ dependencies = [
     "flask>=3.0.0",
     "libcst>=1.1.0",
     "gunicorn>=21.2.0",
+    "typing-extensions>=4.8.0",
```

Comment:
Why do we need this (since we require Python >= 3.12 which is the latest version)

Reply:
`typing-extensions` should always include the latest changes in Python Typing. No harm to explicitly specify it here(in fact it is already installed by other dependents).

BTW, does zeabur non-Dockerfile deployment respect the `requires-python` value?

Reply:
okay I can buy that argument.

> BTW, does zeabur non-Dockerfile deployment respect the requires-python value?
Not sure, I'm asking them